cpp/IceGrid/icebox/README.md

```diff
@@ -1,94 +1,66 @@
 # IceGrid IceBox
 
-This demo shows a very simple [IceGrid deployment][1], with a single
-[IceBox][2] server hosting a number of IceBox services.
+The IceGrid IceBox demo illustrates how to deploy an IceBox server with IceGrid.
 
-The demo also demonstrates how to use an environment variable in a
-server descriptor.
+Follow these steps to build and run the demo:
 
-In `application.xml`, `<env>LANG=en</env>` defines an environment variable
-named `LANG` with value `en`. The Hello object implementation HelloI
-reads the value of `LANG` and shows the message in the corresponding
-language; you can try to update the value to `fr`, `de`, `es` or `it`.
+1. Build the client and server applications:
```

Comment:
We need to update the setup instructions for Windows, until we fix the MSI to add binaries to the PATH..., and also to mention the services come from the MSI. For a follow-up PR.